src/LiveComponent/assets/src/Component/plugins/PollingPlugin.ts

```diff
@@ -12,6 +12,9 @@ export default class implements PluginInterface {
         this.pollingDirector = new PollingDirector(component);
         this.initializePolling();
 
+        // access from stimulus_controller
+        (component as any).pollingDirector = this.pollingDirector;
```

Comment:
Using `any` is a bad practice, if it's really impossible to type the property, then we must use `unknown`.

Here, we know that `component` is type of class `Component` that we define ourselves. So, you must add the property `pollingDirector` on `Component` class instead